pois/generate_pois.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3
+
+import argparse
+import json
+import os
+import subprocess
+
+
+# This tool takes an England-wide osm.pbf (from
+# http://download.geofabrik.de/europe/great-britain/england-latest.osm.pbf) and
+# generates files with point-of-interest layers for ATIP.
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("input", help="Path to england-latest.osm.pbf file", type=str)
+    args = parser.parse_args()
+
+    # https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dschool indicates
```

Comment:
We can share this initial version for feedback to see if universities and other similar facilities should be in the same layer or not